### PR TITLE
raidboss/oopsy: add Worqor Zormor (L93)

### PR DESCRIPTION
**Pull request by wexxlee**

### CODE REVIEW

File: ui/raidboss/data/07-dt/dungeon/worqor-zormor.txt [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+2004.1 "--sync--" StartsUsing { id: "8DBA", source: "Kahderyor" } window 5,5
+2009.1 "Wind Unbound" Ability { id: "8DBA", source: "Kahderyor" }
+2021.6 "Crystalline Crush" Ability { id: "8D39", source: "Kahderyor" }
+2032.7 "Wind Shot" Ability { id: "8DBC", source: "Kahderyor" }
+2041.8 "Earthen Shot" Ability { id: "8DBB", source: "Kahderyor" }
+2051.7 "Crystalline Storm" Ability { id: "8DBE", source: "Kahderyor" }
+2063.0 "Earthen Shot" Ability { id: "8DBB", source: "Kahderyor" }
+2074.1 "Wind Shot" Ability { id: "8DBC", source: "Kahderyor" }
+2084.6 "Wind Unbound" Ability { id: "8DBA", source: "Kahderyor" }
+2093.5 "Seed Crystals" Ability { id: "8DC3", source: "Kahderyor" }
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/dungeon/worqor-zormor.txt:88]: There could be a `spread => break crystals` call for this.

File: ui/raidboss/data/07-dt/dungeon/worqor-zormor.ts [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+          en: 'Spread + Away from lines',
+        },
+      },
+    },
+    {
+      id: 'WorqorZormor Kahderyor Eye of the Fierce',
+      type: 'StartsUsing',
+      netRegex: { id: '8DC9', source: 'Kahderyor', capture: false },
+      response: Responses.lookAway('alert'),
+    },
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/dungeon/worqor-zormor.ts:293]: Could have `out + look away` and `in + look away` for the two variations.

File: ui/raidboss/data/07-dt/dungeon/worqor-zormor.ts [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+      type: 'StartsUsing',
+      netRegex: { id: '8DBC', source: 'Kahderyor', capture: false },
+      infoText: (data, _matches, output) => {
+        if (data.seenCrystallineStorm)
+          return output.stackInLines!();
+        return output.stackInHole!();
+      },
+      outputStrings: {
+        stackInHole: {
+          en: 'Stack donuts in hole',
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/dungeon/worqor-zormor.ts:263]: 😏

File: ui/raidboss/data/07-dt/dungeon/worqor-zormor.ts [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+    },
+    {
+      id: 'WorqorZormor Gurfurlur Wind Tile Followup',
+      type: 'MapEffect',
+      netRegex: { flags: '00020001', location: '1D', capture: false },
+      delaySeconds: 20.4,
+      infoText: (_data, _matches, output) => output.kbAoe2!(),
+      outputStrings: {
+        kbAoe2: {
+          en: 'Knockback + AoE (avoid adds)',
```
Comment (xiashtra) [inline ui/raidboss/data/07-dt/dungeon/worqor-zormor.ts:407]:
```suggestion
          en: 'Knockback + AoE (avoid tornadoes)',
```

File: ui/raidboss/data/07-dt/dungeon/worqor-zormor.ts (hunk 3 above)
Comment (wexxlee) [reply ui/raidboss/data/07-dt/dungeon/worqor-zormor.ts:263]: 😏

**Review by xiashtra — APPROVED** [review]

**Pull request merged**